roles/hoarder/defaults/main.yml

```diff
@@ -0,0 +1,150 @@
+##########################################################################
+# Title:            Sandbox: Hoarder | Default Variables                 #
+# Author(s):        Zuke97                                               #
+# URL:              https://github.com/saltyorg/Sandbox                  #
+# --                                                                     #
+##########################################################################
+#                   GNU General Public License v3.0                      #
+##########################################################################
+---
+################################
+# Basics
+################################
+
+hoarder_name: hoarder
+
+################################
+# Paths
+################################
+
+hoarder_paths_folder: "{{ hoarder_name }}"
+hoarder_paths_location: "{{ server_appdata_path }}/{{ hoarder_paths_folder }}"
+hoarder_paths_folders_list:
+  - "{{ hoarder_paths_location }}"
+
+################################
+# Web
+################################
+
+hoarder_web_subdomain: "{{ hoarder_name }}"
+hoarder_web_domain: "{{ user.domain }}"
+hoarder_web_port: "3000"
+hoarder_web_url: "{{ 'https://' + (hoarder_web_subdomain + '.' + hoarder_web_domain
+                  if (hoarder_web_subdomain | length > 0)
+                  else hoarder_web_domain) }}"
+
+################################
+# DNS
+################################
+
+hoarder_dns_record: "{{ hoarder_web_subdomain }}"
+hoarder_dns_zone: "{{ hoarder_web_domain }}"
+hoarder_dns_proxy: "{{ dns.proxied }}"
+
+################################
+# Traefik
+################################
+
+hoarder_traefik_sso_middleware: ""
+hoarder_traefik_middleware_default: "{{ traefik_default_middleware }}"
+hoarder_traefik_middleware_custom: ""
+hoarder_traefik_certresolver: "{{ traefik_default_certresolver }}"
+hoarder_traefik_enabled: true
+hoarder_traefik_api_enabled: false
+hoarder_traefik_api_endpoint: ""
+
+################################
+# Docker
+################################
+
+# Container
+hoarder_docker_container: "{{ hoarder_name }}"
+
+# Image
+hoarder_docker_image_pull: true
+hoarder_docker_image: "ghcr.io/hoarder-app/hoarder"
+hoarder_docker_image_tag: "release"
+
+# Ports
+hoarder_docker_ports_defaults: []
+hoarder_docker_ports_custom: []
+hoarder_docker_ports: "{{ hoarder_docker_ports_defaults
+                          + hoarder_docker_ports_custom }}"
+
+# Envs
+hoarder_docker_envs_default:
+  TZ: "{{ tz }}"
+  PUID: "{{ uid }}"
+  PGID: "{{ gid }}"
+  HOARDER_VERSION: "release"
+  MEILI_ADDR: "http://meilisearch:7700"
+  BROWSER_WEB_URL: "http://chrome:9222"
```

Comment:
Perhaps these should be exposed as variables in the event someone is doing customizations